index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <title>InstaPizza</title>
+  </head>
+  
+  <body>
+    <h1>&#127829; InstaPizza &#127829;</h1>
+    <div class="button" id="button" onclick="buttonClicked()">
+      <p>The easiest way to order pizza.</p>
+      <p class="call-to-action">Click me to get started!</p>
+    </div>
+    
+    <div class="chat-container" id="chatContainer">
```

Comment:
`id`, same as `class`, should have name in `kebab-case` pattern, not `camelCase`. Don't be afraid to add same name to both `id` and `class` though, it's completely ok :)
Same thing is addressed to lines below

Reply:
leaving this for now and will keep in mind for future projects.